internal/chain/beacon/node.go

```diff
@@ -174,7 +179,7 @@ func (h *Handler) ProcessPartialBeacon(ctx context.Context, p *proto.PartialBeac
 
 	// verify if request is valid
 	span.AddEvent("h.crypto.ThresholdScheme.VerifyPartial")
-	err := h.crypto.ThresholdScheme.VerifyPartial(h.crypto.GetPub(), msg, p.GetPartialSig())
+	err = h.crypto.ThresholdScheme.VerifyPartial(h.crypto.GetPub(), msg, p.GetPartialSig())
 	span.AddEvent("h.crypto.ThresholdScheme.VerifyPartial - done")
```

Comment:
OOS but shouldn't this be only when `err != nil`?

Reply:
the spans are just for tracing, it's fine having an event saying the VerifyPartial (which is done in another library without such tracing in it) is done as soon as it's done IMO.

Reply:
The thing is this might be throwing off your timings, since you'll record timing data for errors along with successful spans. This is OTEL? I think it has error semantics for spans for just this, anyway it's OOS so I guess just bear in mind for future use